src/explicit.rs

```diff
@@ -47,13 +47,17 @@ pub fn compute(
             RLE | LRE | RLO | LRO | RLI | LRI | FSI => {
                 let last_level = stack.last().level;
 
+                // <https://www.unicode.org/reports/tr9/#Retaining_Explicit_Formatting_Characters>
+                levels[i] = last_level;
```

Comment:
I'm not super happy that all this code uses `[]` but you have another issue to follow up on that (fuzzing this crate to make sure it doesn't panic).

Reply:
Yeah the whole crate does this